### ISSUE TITLE
Structlog overwrites sys.stdout through Colorama

### ISSUE THREAD

**Issue by gwtwod** [hynek/structlog#210]
When i import structlog with colorama installed, my sys.stdout is overwritten by colorama.init() called here: https://github.com/hynek/structlog/blob/cc063e0fbc8c395559255c788509a3759f419cec/src/structlog/dev.py#L147

If I then try to force printing of ANSI color codes in my script while piping the results somewhere, all color codes are stripped which I didn't expect.

Could this be avoided somehow? Perhaps by calling colorama.init(wrap=False), or something else if that breaks on windows?

Example to reproduce:
```
#!/usr/bin/env python3
import sys 
print('\033[92m' + 'Before structlog import' + '\033[0m sys.stdout:', sys.stdout)
import structlog
print('\033[92m' + 'After structlog import' + '\033[0m sys.stdout:', sys.stdout)
sys.stdout = sys.__stdout__
print('\033[92m' + 'After resetting' + '\033[0m sys.stdout:', sys.stdout)
```
Example output where 2nd line has no colors anymore:
```
/poc.py | cat
Before structlog import sys.stdout: <_io.TextIOWrapper name='<stdout>' mode='w' encoding='UTF-8'>
After structlog import sys.stdout: <colorama.ansitowin32.StreamWrapper object at 0x7f1c0c981b70>
After resetting sys.stdout: <_io.TextIOWrapper name='<stdout>' mode='w' encoding='UTF-8'>
```

**Comment by hynek** [comment]
Hm, that's very unfortunate indeed. Maybe we should make `colorama.init()` lazy?

**Comment by hynek** [comment]
I hope I fixed it in 3b534afd4f27067502ebacd686cb190c26c6e5f8 – would you mind checking if master fixes the problem for you before I release? TIA
